doc/services/portability/posix/conformance/index.rst

```diff
@@ -0,0 +1,147 @@
+    _POSIX_SEMAPHORES, 200809L, :kconfig:option:`CONFIG_PTHREAD_IPC`
+    _POSIX_SPIN_LOCKS, 200809L, :kconfig:option:`CONFIG_PTHREAD_SPINLOCK`
+    _POSIX_THREAD_SAFE_FUNCTIONS, 200809L,
+    _POSIX_THREADS, -1, :kconfig:option:`CONFIG_PTHREAD_IPC`
+    _POSIX_TIMEOUTS, 200809L, :kconfig:option:`CONFIG_PTHREAD_IPC`
+    _POSIX_TIMERS, -1, :kconfig:option:`CONFIG_POSIX_CLOCK`
+    _POSIX2_C_BIND, 200809L,
+
+.. csv-table:: POSIX System Interfaces (Optional)
+   :header: Symbol, Support, Remarks
+   :widths: 50, 10, 50
+
+    _POSIX_ADVISORY_INFO, -1,
+    _POSIX_CPUTIME, -1,
+    _POSIX_FSYNC, -1,
+    _POSIX_IPV6, 200809L, :kconfig:option:`CONFIG_NET_IPV6`
+    _POSIX_MEMLOCK, -1,
+    _POSIX_MEMLOCK_RANGE, -1,
+    _POSIX_MESSAGE_PASSING, 200809L, :kconfig:option:`CONFIG_POSIX_MQUEUE`
+    _POSIX_MONOTONIC_CLOCK, -1, :kconfig:option:`CONFIG_POSIX_CLOCK`
+    _POSIX_PRIORITIZED_IO, -1,
+    _POSIX_PRIORITY_SCHEDULING, -1,
+    _POSIX_RAW_SOCKETS, 200809L, :kconfig:option:`CONFIG_NET_SOCKETS`
+    _POSIX_SHARED_MEMORY_OBJECTS, -1,
+    _POSIX_SPAWN, -1,
+    _POSIX_SPORADIC_SERVER, -1,
+    _POSIX_SYNCHRONIZED_IO, -1,
+    _POSIX_THREAD_ATTR_STACKADDR, 200809L, :kconfig:option:`CONFIG_PTHREAD`
+    _POSIX_THREAD_CPUTIME, -1,
+    _POSIX_THREAD_ATTR_STACKSIZE, 200809L, :kconfig:option:`CONFIG_PTHREAD`
+    _POSIX_THREAD_PRIO_INHERIT, 200809L, :kconfig:option:`CONFIG_PTHREAD_MUTEX`
+    _POSIX_THREAD_PRIO_PROTECT, -1,
+    _POSIX_THREAD_PRIORITY_SCHEDULING, -1,
+    _POSIX_THREAD_PROCESS_SHARED, -1,
+    _POSIX_THREAD_SPORADIC_SERVER, -1,
+    _POSIX_TRACE, -1,
+    _POSIX_TRACE_EVENT_FILTER, -1,
+    _POSIX_TRACE_INHERIT, -1,
+    _POSIX_TRACE_LOG, -1,
+    _POSIX_TYPED_MEMORY_OBJECTS, -1,
+    _XOPEN_CRYPT, -1,
+    _XOPEN_REALTIME, -1,
+    _XOPEN_REALTIME_THREADS, -1,
+    _XOPEN_STREAMS, -1,
+    _XOPEN_UNIX, -1,
+
+POSIX Shell and Utilities
+=========================
+
+Zephyr does not support a POSIX shell or utilities at this time.
+
+.. csv-table:: POSIX Shell and Utilities
+   :header: Symbol, Support, Remarks
+   :widths: 50, 10, 50
+
+    _POSIX2_C_DEV, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_CHAR_TERM, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_FORT_DEV, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_FORT_RUN, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_LOCALEDEF, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_PBS, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_PBS_ACCOUNTING, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_PBS_LOCATE, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_PBS_MESSAGE, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_PBS_TRACK, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_SW_DEV, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_UPE, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_UNIX, -1, :ref:`†<posix_undefined_behaviour>`
+    _POSIX2_UUCP, -1, :ref:`†<posix_undefined_behaviour>`
+
+XSI Conformance
```

Comment:
should this be in another page?

![Screenshot 2023-10-31 at 10 55 38 AM](https://github.com/zephyrproject-rtos/zephyr/assets/1563974/540a45ea-4d69-4f2e-a09f-c9085a8c298a)